Jropp/jason-ropp-js-iobp
# Onboarding Project: Index.js and my-app unused links

## What It Does

Deletes unused links in index.js and my-app

## To Test

There should be no errors on load
All save, create, edit, delete, sort functionality should work appropriately

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:


## Dependencies

- [ ] #138
